src/SoundList.js

```diff
@@ -57,7 +62,7 @@ const SoundList = ({
                 </span>
               </div>
               <div
-                className={`group-hover:opacity-100 sm:hover:opacity-0 w-6/12 md:w-4/12 md:space-2 hidden sm:flex flex-row justify-end items-center ${
+                className={`group-hover:opacity-100 sm:hover:opacity-0 w-6/12 md:w-5/12 md:space-2 hidden sm:flex flex-row justify-end items-center ${
```

Comment:
We need to change `md:w-4/12` to `md:w-5/12` to give the sound duration more room since they're longer, now.